conf.d/forgit.plugin.fish

```diff
@@ -300,12 +300,17 @@ function forgit::branch::delete -d "git checkout branch deleter"
 end
 
 
-
-function forgit::checkout::branch -d "git checkout branch selector" --argument-names 'input_branch_name'
+# note we "wrap" delete, we just need a snazzy way to get the branch list here
+function forgit::checkout::branch -d "git checkout branch selector" --argument-names 'input_branch_name' --wraps "git branch --delete" 
```

Comment:
Why not use `--wraps "git branch --all"` here, just like it is used for the fzf input? This way completion and fzf window would show the same options, which would be less surprising to me.